model/src/main/java/com/javahelp/model/user/ProviderUserInfo.java

```diff
@@ -98,6 +100,30 @@ public void setPhoneNumber(String phoneNumber) {
         this.phoneNumber = phoneNumber;
     }
 
+    /**
+     * Gets the gender of this provider
+     *
+     * @return the current gender of this client
+     */
+    public String getGender(){
+        return gender;
+    }
+
+    /**
+     * Sets a new gender to this client
+     *
+     * @param gender an updated gender of this client
+     */
+    public String setGender(String gender){
+        if (!(gender.toUpperCase().equals("MALE") || gender.toUpperCase().equals("FEMALE") ||
+                gender.toUpperCase().equals("OTHERS"))){
+            throw new IllegalArgumentException("This is not a valid input. Please try \"MALE\"" +
+                    "or \"FEMALE\" or \"OTHERS\"");
+        }
+        this.gender = gender.toUpperCase();
+        return null;
+    }
```

Comment:
Instead of a `String` it might be a good idea to either make this field some kind of enum type with `MALE`, `FEMALE`, and `OTHER` fields, or to make it a `String` field without restriction. I don't think we should make it a `String` and restrict the possible values.

Reply:
Sure let me do it